### PR TITLE
Implement setting score tag

### PR DESCRIPTION
**Pull request by Enes5519**
Thx for unbanning

## Introduction
You can setting score tag for entities.

### API changes

```
Entity::setScoreTag
Entity::getScoreTag
```

## Tests

```php
// Entity $entity
$entity->setScoreTag("20 <3");
```

### CODE REVIEW

**Comment by Muqsit** [comment]
Are there any in-game/visual testes that you've done?

**Comment by Enes5519** [comment]
Name tags are corrupted but score tags are not corrupted. Strange :thinking:. I am sending screenshoot @Muqsit

**Comment by Enes5519** [comment]
Screenshot_20180727_151652
Screenshot_20180727_151702

**Comment by lukeeey** [comment]
Doesnt that technically mewn you can bypass the nametag "bug" by setting score as nametag?

**Comment by dktapps** [comment]
no, because you still can't change the nametag, only this extra thing underneath it.

**Comment by Enes5519** [comment]
True @dktapps

**Review by dktapps — DISMISSED** [review]
use tabs, not spaces. Other than that, looks good to me.

**Comment by Enes5519** [comment]
@dktapps I write from phpstorm but I am editing now

**Comment by Frago9876543210** [comment]
@dktapps 
> no, because you still can't change the nametag, only this extra thing underneath it.

Bypassed!
default

**Comment by dktapps** [comment]
by setting the username to empty, which has other adverse side effects...

**Comment by Frago9876543210** [comment]
Yet, it's better than nothing

File: src/pocketmine/entity/Entity.php [hunk 1]
```diff
@@ -604,19 +604,19 @@ public function setNameTagAlwaysVisible(bool $value = true) : void{
...
-
-    /**
-     * @param string $score
-     */
-    public function setScoreTag(string $score) : void{
-	    $this->propertyManager->setString(self::DATA_SCORE_TAG, $score);
-    }
+	/**
+	 * @return null|string
+ 	 */
```
Comment (dktapps) [inline src/pocketmine/entity/Entity.php]: there's still spaces here

**Comment by Enes5519** [comment]
Are you gonna merge this pr? @dktapps

**Comment by Enes5519** [comment]
@Frago9876543210 Score tags doesn't seem from a distant place.

**Review by jasonw4331 — APPROVED** [review]

**Comment by Ammarking** [comment]
To be honest having the name tag hidden until you get close to a player has positive effects too, I think having a score tag is a great idea.

**Comment by dktapps** [comment]
Squash-merged into `release/3.2` and `master` as 99045fe21aea2b4c4d6fd426e71763ad4b2571f3.

**Pull request closed without merging**